mlflow/pipelines/cards/__init__.py

```diff
@@ -0,0 +1,291 @@
+from __future__ import annotations
+
+import re
+import base64
+import html
+import os
+import pathlib
+import logging
+import random
+import string
+from io import StringIO
+from typing import Union
+
+from packaging.version import Version
+
+from mlflow.exceptions import MlflowException, INVALID_PARAMETER_VALUE
+
+CARD_PICKLE_NAME = "card.pkl"
+CARD_HTML_NAME = "card.html"
+
+# TODO: Make card save / load including card_resources directory
+_CARD_RESOURCE_DIR_NAME = f"{CARD_HTML_NAME}.resources"
```

Comment:
@WeichenXu123 Can we remove this variable?

Reply:
yes. we can remove it.